README.md

```diff
@@ -49,107 +44,62 @@ Or simply load the SDK by importing the script into the header of your HTML file
-```javascript
-auth.customGrant({
-    attachToken: false,
-    data: {
-        client_id: "{{clientId}}",
-        grant_type: "account_switch",
-        scope: "{{scope}}",
-        token: "{{token}}",
-    },
-    id: "account-switch",
-    returnResponse: true,
-    returnsSession: true,
-    signInRequired: true
-});
 ```
 
-[Learn more](#customgrant).
-
-### Callback Hooks
-Callback functions can be attached to authentication methods such as sign in and sign out using hooks. Hooks can be attached to the client using the `on()` method.
-```javascript
- auth.on("sign-in", (response) => {
-     alert("You have successfully signed in!");
- });
-```
-[Learn more](#on).
+[Learn more](#apis).
 
-## 3. Try Out the Sample Apps
-### Create a Service Provider
-Before trying out the sample apps, you need to a create a service provider in the Identity Server. So, navigate to `https://localhost:9443/carbon" and click on `Add` under `Service Providers` in the left-hand menu panel.
+## Try Out the Sample Apps
+### 1. Create a Service Provider
+Before trying out the sample apps, you need to a create a service provider in the Identity Server.
+1. So, navigate to `https://localhost:9443/carbon" and click on `Add` under `Service Providers` in the left-hand menu panel.
 
-Enter `Sample` as the name of the app and click on `Register`.
+2. Enter `Sample` as the name of the app and click on `Register`.
 
-Then, expand the `Inbound Authentication Configuration` section. Under that, expand `OAuth/OpenID Connect Configuration` section and click on `Configure`.
+3. Then, expand the `Inbound Authentication Configuration` section. Under that, expand `OAuth/OpenID Connect Configuration` section and click on `Configure`.
 
-Under `Allowed Grant Types` uncheck everything except `Code` and `Refresh Token`.
+4. Under `Allowed Grant Types` uncheck everything except `Code` and `Refresh Token`.
 
-Enter `http://localhost:3000` as the `Callback Url`.
+5. Enter `http://localhost:3000` as the `Callback Url`.
 
-Check `Allow authentication without the client secret`.
+6. Check `Allow authentication without the client secret`.
 
-Click `Add` at the bottom.
+7. Click `Add` at the bottom.
 
-Copy the `OAuth Client Key`.
+8. Copy the `OAuth Client Key`.
 
-### Running the sample apps
+### 2. Running the sample apps
 Build the apps by running the following command at the root directory.
 ```
 npm run build
 ```
 
-### Vanilla JavaScript Sample
+### 3. Vanilla JavaScript Sample
 You can try out the Vanilla JavaScript Sample App from the [oidc-sample-apps/vanilla-js-app](oidc-sample-apps/vanilla-js-app). The instructions to run the app can  be found [here](oidc-sample-apps/vanilla-js-app/README.md)
 
-## 4. APIs
+## APIs
 ### getInstance
 This returns an instance of the `IdentityClient`. Since the `IdentityClient` is a singleton, this method returns the same instance no matter how many time sit is called.
```

Comment:
```suggestion
This returns an instance of the `IdentityClient`. Since the `IdentityClient` is a singleton, this method returns the same instance no matter how many times it is called.
```